crates/primitives/src/env.rs

```diff
@@ -8,8 +8,7 @@ use crate::{
     VERSIONED_HASH_VERSION_KZG,
 };
 use core::cmp::{min, Ordering};
-use std::boxed::Box;
```

Comment:
can we reduce such changes, it will make upcoming merge with revm more difficulty.

Reply:
Fixed